app/helpers/fuzzy_time.rb

```diff
@@ -22,6 +22,8 @@ def ago(timestamp)
                        "about an hour and a half ago"
                      when less_than(23.5 * hours)
                        "about #{(diff / (1 * hours)).round} hours ago"
+                     when less_than(24 * hours)
+                       "about 23.5 to 24 hours ago"
                      end
```

Comment:
Is 23.5 to 24 hours really special enough to require its own case?

Why can't it just be "about 24 hours ago"?